### ISSUE TITLE
Getting beacon to work when switching window configuration with eye-browse package

### ISSUE THREAD

**Issue by JSpenced** [Malabarba/beacon#66]
If I switch between different window configurations using the eye-browse package. The beacon isn't activated. Is there a way to include the eye-browse commands to initiate beacon when the different configurations are selected?

**Comment by Malabarba** [comment]
Does eye browse have hooks? If so, it's just a matter of adding beacon-blink to something like eye-browse-after-change-configuration-hook.

If it doesn't have a hook like that then you can still use an advice.

**Comment by JSpenced** [comment]
I added in:
```
(add-hook 'eyebrowse-post-window-switch-hook (lambda () (interactive) (beacon-blink)))
```
but it doesn't seem to work. Is this the correct use of `beacon-blink`. Maybe some issue with the hook though.

**Comment by Malabarba** [comment]
Just doing `(add-hook 'eyebrowse-post-window-switch-hook #'beacon-blink)` should be enough.

If that doesn't work, it might be because eyebrowse still does some form of window/buffer management after running the hook. There are a few events that will cause beacon to abort blinking (such as changing buffer or window).

**Comment by JSpenced** [comment]
Yeah, I am not sure what is happening but that doesn't work. Even defining a function then calling `(beacon-blink)` doesn't work. Like shown below for example but beacon-blink still doesn't run:

```
(defun eyebrowse-switch-to-window-config-22 ()
  "Switch to window configuration 22."
  (interactive)
  (eyebrowse-switch-to-window-config 22)
  (beacon-blink))
```

**Comment by Malabarba** [comment]
What happens if you run the following code 

```
(defun test-blinking ()
  (interactive)
  (display-buffer (car (last (buffer-list))))
  (beacon-blink))
```

and then call `M-x test-blinking`?

**Comment by JSpenced** [comment]
Yeah, that works fine! All the other functions of beacon-blink seem to work fine as well. Like on emacs gaining focus, etc...

**Comment by Malabarba** [comment]
I guess it's something specific to how `  (eyebrowse-switch-to-window-config 22)` works then. It's hard to figure out without digging deep into the package code.

**Comment by JSpenced** [comment]
No worries, just thought if it was possible that would be great.
